res/openvpn-gui-res-it.rc

```diff
@@ -411,8 +448,30 @@ BEGIN
     IDS_ERR_DISCONN_SCRIPT_TIMEOUT "Il valore di registro ""disconnectscript_timeout"" deve essere un numero tra 1 and 99."
     IDS_ERR_PRECONN_SCRIPT_TIMEOUT "Il valore di registro ""preconnectscript_timeout"" deve essere un numero tra 1 and 99."
     IDS_ERR_CREATE_REG_KEY "Errore nella creazione della chiave HKLM\\SOFTWARE\\OpenVPN-GUI."
-    IDS_ERR_OPEN_WRITE_REG "Fallita l'apertura del registro in scrittura. Puoi far partire questa applicazione " \
+    IDS_ERR_OPEN_WRITE_REG "Errore nell'apertura del registro in scrittura. Puoi far partire questa applicazione " \
                            "una volta che l'amministratore ha aggiornato il registro."
-    IDS_ERR_READ_SET_KEY "Errore nella lettura e settaggio della chiave di registro ""%s""."
+    IDS_ERR_READ_SET_KEY "Errore nella lettura e modifica della chiave di registro ""%s""."
     IDS_ERR_WRITE_REGVALUE "Errore nella scrittura del valore di registro ""HKEY_CURRENT_USER\\%s\\%s""."
+    
+    /* importation */
+    IDS_ERR_IMPORT_EXISTS "Esiste già una configurazione con il nome ""%s"" already exists."
```

Comment:
I guess you wanted to remove "already exists" at the end :)

Reply:
Ouch.